src/NzbDrone.Core/Datastore/Migration/066_add_tags.cs

```diff
@@ -16,6 +16,9 @@ protected override void MainDbUpgrade()
 
             Alter.Table("Notifications")
                  .AddColumn("Tags").AsString().Nullable();
+
+            Execute.Sql("UPDATE Series SET Tags = '[]'");
+            Execute.Sql("UPDATE Notifications SET Tags = '[]'");
```

Comment:
I'm not sure about these, we should update `EmbededDocumentConverter` to deal with these properly. null should be treated the same as empty list when target is an `IEnumarable`